site/source/_posts/2018/01/17-Announcing-Dojo-2-beta-5/Announcing-Dojo-2-beta-5.md

```diff
@@ -0,0 +1,109 @@
+
+If you're learning about Dojo 2 for the first time, please read our overview about the [Dojo 2 beta 4 release](https://dojo.io/blog/2017/12/04/Announcing-Dojo-2-beta-4/).
+
+{% asset_img featured.png feature-full %}
+
+<!-- more -->
+
+## Getting Beta 5
+
+Beta 5 is currently tagged as `latest` in npm, so simply updating to the latest version will provide you with the Beta 5 release.  We also have a release tag of `beta5` in npm if you want to focus just on using Beta 5.
+
+This will be the final beta release for Dojo 2! As we make releases of packages in anticipation of Release Candidate 1, we will not publish those packages to `latest` until Release Candidate 1 is available.
+
+## Just getting started
+
+If you want to get started with Dojo 2, the best way is to take a look at our [tutorials](/tutorials/) or even take a look at Dojo 2 in action with our [example applications](https://dojo.github.io/examples/) (source on [GitHub](https://github.com/dojo/examples)).  Good starting tutorials are:
+
+* [Dojo 2 local installation](/tutorials/000_local_installation/)
+* [Your first Dojo 2 application](/tutorials/001_static_content/)
+* [Creating widgets](/tutorials/003_creating_widgets/)
+
+## New in beta 5
+
+Since the beta 4 release, we have made a number of improvements. The biggest changes and additions are mentioned here, with a complete list of changes
+
+### Initial re-architecture of the build system
+
+Prior to beta5, the structure of the build commands was focused around the tooling and not around the actual creation of applications. So we completed a re-architecture and restructuring of the commands to align more to their use. Specifically, `@dojo/cli-build-webpack` is deprecated and replaced with:
+
+* [`@dojo/cli-build-app`](https://github.com/dojo/cli-build-app/) A command that builds an application
+* [`@dojo/cli-build-widget`](https://github.com/dojo/cli-build-widget/) A command that build a widget, including support for web components
+* Other commands may be planned for the future
+
+This refactoring is mostly complete for Beta 5, with a few remaining items to complete for release candidate 1.
+
+Usage of the build system is reflected in updates to the [Dojo 2 tutorials](https://dojo.io/tutorials/).
+
+### Improvements to using Custom Elements/Web Components
+
+TODO: Explain
+
+### widget-core performance and memory improvements
+
+TODO: Explain, provide metrics?
+
+### New widgets
+
+Two new widgets were added in Beta 5:
+
+* [Progress](https://github.com/dojo/widgets/issues/385) (`@dojo/widgets/progress/Progress`)
+* [Toolbar](https://github.com/dojo/widgets/issues/386) (`@dojo/widgets/toolbar/Toolbar`)
+
+TODO Add screenshots
+
+These widgets may be viewed in the [widget showcase](https://dojo.github.io/examples/widget-showcase/).  TODO: blocked by https://github.com/dojo/examples/issues/278
+
+### Stores
+
+A number of refinements were made since the initial `@dojo/stores` release in Beta 4:
```

Comment:
Stores wasn't really officially part of the `beta4` release set (not tagged), perhaps we can add some more noise around this?

Reply:
They were featured in the beta4 blog... https://dojo.io/blog/2017/12/04/Announcing-Dojo-2-beta-4/ so I didn't want to repeat here.

Reply:
Okay but stores usability has been enhanced quite nicely, worth perhaps pointing out the typings for state and payload.

Reply:
Ok, added

Reply:
Done.